app/controllers/api/v1/apps_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Api
+  module V1
+    class AppsController < ApplicationController
+      def index
+        search = App.where('title ilike ?', "%#{params[:title]}%")
+        render json: search.to_json(only: %i[title apple_url android_url icon_url])
```

Comment:
потом все превратится вот в такую кашу
```suggestion
        render json: search.to_json( :include => { :posts => {
                                 :include => { :comments => {
                                               :only => :body } },
                                 :only => :title } }))
```

Reply:
точно ли only: %i[title apple_url android_url icon_url] добавляет `SELECT title, apple_url, android_url, icon_url ...` в запрос? если нет - это точно плохое решение, но не самое худшее.
кажется  у вас уже где-то использовались сериалайзеры. заюзай их)